python/doc/user_manual/response_surface/low_rank_approximation.rst

```diff
@@ -1,18 +0,0 @@
-========================================
-Response surface: low-rank approximation
-========================================
-
-.. currentmodule:: openturns
-
-.. autosummary::
-    :toctree: _generated/
-    :template: class.rst_t
-
-    CanonicalTensorEvaluation
-    CanonicalTensorGradient
-
-    :template: classWithPlot.rst_t
-    TensorApproximationAlgorithm
```

Comment:
I don't think you removed src/TensorApproximationAlgorithm_doc.i.in or the other low-rank approximation-related API doc files. Is there a reason why? Do we need to wait until the classes are removed, not just deprecated?

Reply:
yes, lets remove all file at once